audio/mp3val/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           makefile 1.0
+
+name                mp3val
+version             0.1.8
+revision            0
+categories          audio
+license             GPL-2
+platforms           darwin
+
+maintainers         {@catap korins.ky:kirill} openmaintainer
+
+description         mp3val is a small, high-speed, free software tool for checking MPEG audio files' integrity.
+long_description    MP3val is a small, high-speed, free software tool for checking MPEG audio files' integrity. \
+                    It can be useful for finding corrupted files (e.g. incompletely downloaded, truncated, containing garbage). \
+                    MP3val is also able to fix most of the problems. Being a multiplatform application, \
+                    MP3val can be runned both under Windows and under Linux (or BSD).
+
+homepage            http://mp3val.sourceforge.net/
+
+master_sites        sourceforge:project/mp3val/mp3val/mp3val%20${version}
+distname            mp3val-${version}-src
+
+checksums           rmd160  25b63be18d40623324b1ce4d622bd18204965c13 \
+                    sha256  95a16efe3c352bb31d23d68ee5cb8bb8ebd9868d3dcf0d84c96864f80c31c39f \
+                    size    21406
+
+build.target        mp3val
+build.args          -f Makefile.linux
+
+makefile.has_destdir no
+
+patchfiles          Makefile-cxxflags.patch
+
+destroot {
+    xinstall -m 755 ${worksrcpath}/mp3val ${destroot}${prefix}/bin
+
+    xinstall -d ${destroot}${prefix}/share/doc/mp3_check
+
+    foreach file {COPYING manual.html changelog.txt} {
+        xinstall -m 0644 ${worksrcpath}/${file} ${destroot}${prefix}/share/doc/mp3val
```

Comment:
this doesn't do anything as you can see when inspecting the installed files:
```
% port contents mp3val
Port mp3val contains:
  /opt/local/bin/mp3val
  /opt/local/share/doc/mp3val
```

I guess you meant to do:
```
    set docdir ${prefix}/share/doc/${name}
    xinstall -d ${destroot}${docdir}
    xinstall -m 0644 -W ${worksrcpath} COPYING manual.html changelog.txt \
        ${destroot}${docdir}
```